File: 05. Percabangan/G. Jarak Manhattan/g_jarak_manhattan.c
#include <stdio.h>
int main(){
  int x1, y1, x2, y2;
  scanf("%d %d %d %d", &x1, &y1, &x2, &y2);
  x1 = x1 - x2;
  y1 = y1 - y2;
  if (x1 < 0) x1 = x1 * -1;
  if (y1 < 0) y1 = y1 * -1;
  printf("%d\n", x1 + y1);
  return 0;
}

File: 05. Percabangan/G. Jarak Manhattan/g_jarak_manhattan.go
package main

import (
  "fmt"
)

func main() {
  var x1, y1, x2, y2 int32
  fmt.Scanf("%d %d %d %d", &x1, &y1, &x2, &y2)
  x1 = x1 - x2
  y1 = y1 - y2
  if x1 < 0 {
    x1 = x1 * -1
  }
  if y1 < 0 {
    y1 = y1 * -1
  }
  fmt.Println(x1 + y1)
}
